### PR TITLE
RA-545: TimeNode returns a default start/end time if Model is null

### PR DESCRIPTION
**Pull request by aameijide**

### CODE REVIEW

File: VAS.Core/ViewModel/TimeNodeVM.cs [hunk 1]
```diff
@@ -50,7 +50,7 @@ public TimeNodeVM ()
 		/// </summary>
 		public Time Start {
 			get {
-				return Model.Start;
+				return Model?.Start ?? new Time (0);
```
Comment (ylatuya) [inline VAS.Core/ViewModel/TimeNodeVM.cs]: I would handle that diferently. In the MediaFileSetVM if model == null, VisibleRegion.Model = new TimeNode (0, 0)

**Comment by ylatuya** [comment]
Also, missing unit tests.

**Comment by aameijide** [comment]
About missing test, I don't think that a unit test to check if is null return (or sets in your approach) 0 is worthy.

**Comment by ylatuya** [comment]
As I commented in the code, this is not the correct fix, it's in MediaFileSetVM. It always make sense to do write test, specially if, like in this case, it was causing a bug.
